entity_gym/examples/slider.py

```diff
@@ -0,0 +1,82 @@
+import random
+from dataclasses import dataclass
+from typing import Dict, Mapping
+
+import numpy as np
+
+from entity_gym.env import (
+    Action,
+    ActionSpace,
+    CategoricalAction,
+    CategoricalActionMask,
+    CategoricalActionSpace,
+    Entity,
+    Environment,
+    Observation,
+    ObsSpace,
+)
+
+
+@dataclass
+class Slider(Environment):
+    """
+    On each timestep, there is either a generic "Object" entity with a `is_hotdog` property, or a "Hotdog" object.
+    The "Player" entity is always present, and has an action to classify the other entity as hotdog or not hotdog.
+    """
```

Comment:
could probably unify the slider and continuous slider environments by passing in a bool that determines whether we use continuous or discrete action